tests/gravity_point_values.prm

```diff
@@ -0,0 +1,69 @@
+# A simple setup for gravity acceleration calculated at a point P
+# manual for more information.
```

Comment:
at a point P manual for more information 
-> 
at a point P. See the corresponding section in the manual for more information.